### PR TITLE
Tap8

### PR DESCRIPTION
**Pull request by mnm678**
**Fixes issue #**: Tap 8

**Description of the changes being introduced by the pull request**:
Draft of my implementation of Tap 8 for review, comments, etc.

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: tuf/repository_tool.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import tempfile
 import shutil
 import json
+import hashlib
```
Comment (trishankatdatadog) [inline tuf/repository_tool.py]: Why use `hashlib` directly instead of `securesystemslib`?

Comment (mnm678) [reply tuf/repository_tool.py]: Good point. I updated this in the most recent commit.

Comment (trishankatdatadog) [reply tuf/repository_tool.py]: Thanks!

**Comment by lukpueh** [comment]
@mnm678, how should we proceed with this PR? It needs a lot of clean up to get merged, and I'd rather have us focus our efforts on the refactor (https://github.com/theupdateframework/tuf/projects/2).

Are you fine with closing here and submitting a new TAP8 POC once we have TUF 1.0.0?

**Comment by joshuagl** [comment]
> Are you fine with closing here and submitting a new TAP8 POC once we have TUF 1.0.0?

In a recent TAP editors meeting I had volunteered to PoC TAP 8. I was planning to do it on top of the refactor branch. I mention this here so that we can coordinate, and not duplicate, efforts.

**Comment by mnm678** [comment]
> Are you fine with closing here and submitting a new TAP8 POC once we have TUF 1.0.0?

Yes. I think the implementation could be a bit cleaner after the refactor anyway.

> In a recent TAP editors meeting I had volunteered to PoC TAP 8. I was planning to do it on top of the refactor branch. I mention this here so that we can coordinate, and not duplicate, efforts.

That would be great! Let me know when you're ready to coordinate on that. This can certainly wait until the refactor is wrapping up.

**Pull request closed without merging**